runtime/calamari/Cargo.toml

```diff
@@ -220,3 +221,11 @@ std = [
   'orml-traits/std',
   'orml-xtokens/std',
 ]
+
+# default any debug printing from wasm execution is stripped for generating runtime size as small as possible.
+# if use `force-debug` feature, the debug printing will be open.
+force-debug = ['sp-debug-derive/force-debug']
+# A feature that should be enabled when the runtime should be build for on-chain
+# deployment. This will disable stuff that shouldn't be part of the on-chain wasm
+# to make it smaller like logging for example.
+on-chain-release-build = [ "sp-api/disable-logging" ]
```

Comment:
Did you get a reading on how much the size shrinks ?

Also please add this to the release workflow https://github.com/Manta-Network/Manta/blob/manta/.github/workflows/publish_draft_releases.yml#L134

Reply:
compile size[without on-chain-release-build]
manta size: 79MB
calamari_runtime.compact.compressed.wasm: 692K
dolphin_runtime.compact.compressed.wasm: 898K
kusama_runtime.compact.compressed.wasm: 1.2M
manta_runtime.compact.compressed.wasm: 543K
polkadot_runtime.compact.compressed.wasm: 1.1M

[with on-chain-release-build]
manta size: 77MB
calamari_runtime.compact.compressed.wasm: 666K
dolphin_runtime.compact.compressed.wasm: 875K
kusama_runtime.compact.compressed.wasm: 1.2M
manta_runtime.compact.compressed.wasm: 521K
polkadot_runtime.compact.compressed.wasm: 1.1M

@ghzlatarev

Reply:
Meh, better than. nothing. Might as well include it.

Reply:
https://github.com/paritytech/substrate/blob/master/primitives/api/src/lib.rs#L44-L54 fyi